cmake/configure_linux.gnu.cmake

```diff
@@ -0,0 +1,35 @@
+message("")
```

Comment:
We will need a cmake/configure_macos.gnu.cmake as well.

Reply:
Committed in d62d01a